vsg/rules/process/rule_600.py

```diff
@@ -0,0 +1,19 @@
+
+from vsg.rules import token_suffix
+
+from vsg import token
+
+lTokens = []
+lTokens.append(token.process_statement.process_label)
+lTokens.append(token.process_statement.end_process_label)
+
+
+class rule_600(token_suffix):
+    '''
+    Constant rule 600 checks for suffixes in process labels.
+    '''
+
+    def __init__(self):
+        token_suffix.__init__(self, 'process', '600', lTokens)
+        self.suffixes = ['_proc']
+        self.solution = 'Process labels'
```

Comment:
I was going to ask you exactly what these lines do in each rule, as I'm not confident they are consistent across the new rules I added, and some of the existing rule .e.g some do not have `solution`

Reply:
Maybe you can review these lines in each rule to double check?

Reply:
I reviewed the `token_suffix.py` file and `self.solution` is not used.  I was at one point, but line 51 generates the solution based on the token being suffixed and the possible suffixes defined the by rule.  I must not have cleaned up the other rules.

Reply:
The suffix rule uses `self.suffixes` in its checks.